plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java

```diff
@@ -2032,13 +2065,19 @@ public void registerCoreMembers() {
         // @Group entity
         //
         // @Description
-        // TODO: Document Command Details
+        // For non-armor stand entities, applies a maximum duration invisibility potion effect. (2,147,483,647t, or about 3.4 years)

```

Comment:
`Toggles the invisibility state of an entity` is probably sufficient for this description

Reply:
It also does not just apply a potion with duration - I'm not sure all cases, but at least NPC's have it continually retained via an Invisible trait that is added and enabled.